images/base-emu/Dockerfile

```diff
@@ -54,70 +54,44 @@ RUN \
         apt-get remove -y python3-six && \
 		apt-get autoremove -y && \
 		rm -rf /var/lib/apt/lists/*
-		
-# Downloading PCSX2 AppImage
-RUN \
-    echo "**** Downloading PCSX2 AppImage ****" && \
-        mkdir -p /tmp && \
-        cd /tmp && \
-		curl -H "X-GitHub-Api-Version: "$(GITHUB_REST_VERSION)"" \
-        https://api.github.com/repos/PCSX2/pcsx2/releases/latest | \
-        jq '.assets[]|select(.name|endswith(".AppImage"))'.browser_download_url | \
-        xargs wget -O pcsx2-emu-Qt.AppImage 
-
-# Downloading XEMU AppImage
-RUN \
-    echo "**** Downloading XEMU AppImage ****" && \
-        mkdir -p /tmp && \
-        cd /tmp && \
-		curl -H "X-GitHub-Api-Version: "$(GITHUB_REST_VERSION)"" \
-        https://api.github.com/repos/xemu-project/xemu/releases/latest | \
-        jq '.assets[]|select(.name|endswith(".AppImage"))|select(.name|contains("dbg")|not).browser_download_url' | \
```

Comment:
Is this `|select(.name|contains("dbg")|not).browser_download_url'` part not needed to get `XEMU`?

Reply:
Nice catch! Fixed now